.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 script:
 - npm run test-travis
 - npm run lint
-- npm run pkg
+- npm run pkg && mv bin packages/validator
```

Comment:
I think that all that went wrong last time is that the `.releaserc` file is moved to the individual packages for release. So, the `bin` folder wasn't present for the validator package, which we want to receive the binaries. Hopefully this works 🤞